### ISSUE TITLE
Sorting by Lifecycle, startup time

### ISSUE THREAD

**Issue by alexmucde** [COVESA/dlt-viewer#307]
Currently sorting by timestamp is only supported for a whole file, but not for dlt files containing several lifecycle.
Also startup time since the beginning of a lifecycle should be calculated and shown.

Feedback from Linkedin.

**Comment by eaj40** [comment]
Can i work on this

**Comment by alexmucde** [comment]
@eaj40 This would be great

**Comment by eaj40** [comment]
Documentation for feature, page number in dlt viewer user manuual

**Comment by eaj40** [comment]
Run dlt viewer on win11 pc and run dlt daemon on virtual box Ubuntu connect over TCP
Is this possible

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by alexmucde** [comment]
Yes this would help, then you can restart the VM for several Lifecycles.

**Comment by eaj40** [comment]
Steps to reproduce the problem

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows

________________________________
From: joe jose ***@***.***>
Sent: Monday, April 8, 2024 8:01:34 PM
To: COVESA/dlt-viewer ***@***.***>; COVESA/dlt-viewer ***@***.***>
Cc: Mention ***@***.***>
Subject: RE: [COVESA/dlt-viewer] Sorting by Lifecycle, startup time (Issue #307)


How  to take traces for the problem



Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows



________________________________

**Comment by eaj40** [comment]
What do you mean by startup time

**Comment by eaj40** [comment]
Solution ready, next steps

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows

________________________________
From: joe jose ***@***.***>
Sent: Monday, April 8, 2024 8:17:23 PM
To: COVESA/dlt-viewer ***@***.***>; COVESA/dlt-viewer ***@***.***>
Cc: Mention ***@***.***>
Subject: RE: [COVESA/dlt-viewer] Sorting by Lifecycle, startup time (Issue #307)


Steps to reproduce the problem



Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows



________________________________
From: joe jose ***@***.***>
Sent: Monday, April 8, 2024 8:01:34 PM
To: COVESA/dlt-viewer ***@***.***>; COVESA/dlt-viewer ***@***.***>
Cc: Mention ***@***.***>
Subject: RE: [COVESA/dlt-viewer] Sorting by Lifecycle, startup time (Issue #307)


How  to take traces for the problem



Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows



________________________________
